src/main/java/ua/tc/marketplace/controller/DemoController.java

```diff
@@ -27,4 +28,10 @@ public String demo() {
   public ResponseEntity<Page<DemoRequest>> getAllDemo(@PageableDefault Pageable pageable) {
```

Comment:
Are we able to customize paging parameters? I suggest receiving url parameters pageNumber and pageSize and creating Pageble in service method